tools/issue-labeler/src/IssueLabelerService/Answers/OpenAiAnswerService.cs

```diff
@@ -37,37 +37,69 @@ public async Task<AnswerOutput> AnswerQuery(IssuePayload issue, Dictionary<strin
             // Query + Filtering configurations
             string query = $"{issue.Title} {issue.Body}";
             int top = int.Parse(_config.SourceCount);
-            double scoreThreshold = double.Parse(_config.ScoreThreshold);
-            double solutionThreshold = double.Parse(_config.SolutionThreshold);
+            var scoreThreshold = double.Parse(_config.ScoreThreshold);
+            var solutionThreshold = double.Parse(_config.SolutionThreshold);
+
+            //Subquery generation configurations
+            int minSubqueries = 3;
+            int maxSubqueries = 7;
+            int charsPerSubquery = 30;
```

Comment:
```suggestion
            const int MinSubqueries = 3;
            const int MaxSubqueries = 7;
            const int CharsPerSubquery = 30;
```

Let's also hoist these up to the class level, please.